lib/python/ray/graph.py

```diff
@@ -0,0 +1,49 @@
+# Utilities to deal with computation graphs
+
+import ray.internal.graph_pb2
+import graphviz
```

Comment:
`graphviz` needs to be added to the dependencies. I just tried `python shell.py` inside of `ray/scripts/`, and I got
```
Traceback (most recent call last):
  File "shell.py", line 5, in <module>
    import ray
  File "/Users/rkn/Workspace/ray/lib/python/ray/__init__.py", line 14, in <module>
    from graph import graph_to_dot
  File "/Users/rkn/Workspace/ray/lib/python/ray/graph.py", line 4, in <module>
    import graphviz
ImportError: No module named graphviz
```